src-worker/runner.ts

```diff
@@ -124,7 +130,12 @@ export class Runner {
           console.error(e)
         }
 
-        this.eventSource.emit('attack', { type: 'atack', url: target.site.page, log: `${target.site.page} | ${code}` })
+        this.eventSource.emit('attack', {
+          type: 'atack',
```

Comment:
I know it's more to change, but ` type: 'atack',` => `attack` - I assume it's widely used type, so you can ignore my comment. Sure thing it's extremely low priority these days. That would be refactoring.